styles/app.scss

```diff
@@ -4,5 +4,25 @@
 $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
+@import 'nav';
+@import 'hero';
+@import 'main';
+@import 'footer';
 
-@import 'adonis';
+@media only screen
+  and (min-device-width: 768px)
+  and (max-device-width: 1024px) {
+    .full-page  {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+@media only screen
+  and (min-device-width: 400px)
+  and (max-device-width: 767px) {
+    .full-page  {
+      display: flex;
+      flex-direction: column;
+      flex-wrap: wrap;n
```

Comment:
The `n` at the end of this line is causing a syntax error. Please fix this and ping for further review.